dahee/app.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express')
+const cors = require('cors')
+const app = express()
+const dotenv = require('dotenv')
+
+dotenv.config()
```

Comment:
```suggestion
require('dotenv').config();
```
1. 환경변수를 읽어올 수 있도록 작성되는 코드는 파일 내 최상단에 작성되면 좋을 것 같습니다. 물론 환경변수를 사용하기 전에만 작성되면 동작하는데 문제 없긴 합니다.
2. 기존처럼 두줄로 작성되어 있던 코드는 위와같이 한줄로 줄일 수 있습니다.